build/build-bundle.js

```diff
@@ -38,7 +38,8 @@ const isDevtools = file => path.basename(file).includes('devtools');
 /** @param {string} file */
 const isLightrider = file => path.basename(file).includes('lightrider');
 
-const BANNER = `// lighthouse, browserified. ${VERSION} (${COMMIT_HASH})\n`;
+const BANNER = `// lighthouse, browserified. ${VERSION} (${COMMIT_HASH})\n` +
+  '// @ts-nocheck\n'; // To prevent tsc stepping into any required bundles.
```

Comment:
can we just exclude the bundle in tsconfig?

Reply:
> can we just exclude the bundle in tsconfig?

from my experience this isn't possible (include and exclude are just including and excluding *root* files that tsc starts walking from), but I would be happy if I was missing something :)

See https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#details ("Any files that are referenced by files included...")